webrtc-stats.html

```diff
@@ -244,18 +241,15 @@
       </section>
       <section>
         <h3>
-          Lifetime of stats objects
+          Stats objects for deleted monitored objects
         </h3>
         <p>
-          The general rule is that stats objects are considered to be created at the same time as
-          the underlying object they report on, and once created, they exist for the lifetime of
-          the PeerConnection that contains them, even when the underlying object is stopped or
-          deleted. This is important in order to report consistently on short-lived objects and to
-          be able to report totals over the lifetime of a PeerConnection.
-        </p>
-        <p>
-          When an object is closed or deleted, the <code>timestamp</code> member of the stats
-          object stops updating; it is frozen at the time when the object stopped.
+          When a monitored object is destroyed, a final stats object is produced, carrying the
+          values current at the time of destruction. This object will be returned, with a timestamp
+          reflecting the time of destruction, whenever getStats() is called, as long as the
+          PeerConnection exists. This is important in order to report consistently on short-lived
+          objects and to be able to consistently report totals over the lifetime of a
+          PeerConnection.
```

Comment:
There are two proposals here:
1. that the getStats returns for an object that was destroyed
2. that returned object contains a snapshot of the values at the time of destruction.

If my memory serves me correctly, I believe that we did agree on 1. I cannot say we explicitly discussed 2. Although I would have expected that particular behavior.